kernel/names.mli

```diff
@@ -513,10 +513,39 @@ end
 
 type inductive = Ind.t
 
-(** Designation of a (particular) constructor of a (particular) inductive type. *)
-type constructor = inductive   (* designates the inductive type *)
-                 * int         (* the index of the constructor
-                                  BEWARE: indexing starts from 1. *)
+module Construct :
```

Comment:
Why not Constructor?

Reply:
The rationale was to follow the name of the constructors for kind_of_constr.